tests/monitoring_test.go

```diff
@@ -301,7 +306,7 @@ func checkAlert(alertName string) (*promApiv1.Alert, error) {
 }
 
 func waitForAlertToActivate(alertName string) {
-	Eventually(func() error {
+	EventuallyWithOffset(1, func() error {
```

Comment:
Is this necessary for the actual fix?

Reply:
OK, can you at least move these changes into a separate commit please?

Reply:
done!